s4/clarity/project.py

```diff
@@ -15,3 +16,9 @@ class Project(FieldsMixin, ClarityElement):
     close_date = subnode_property("close-date", types.DATE)
     invoice_date = subnode_property("invoice-date", types.DATE)
     researcher = subnode_link(Researcher, "researcher", attributes=('uri',))  # type: Researcher
+
+    @property
+    def files(self):
```

Comment:
Any reason you didn't use `files = subnode_links(File, File.UNIVERSAL_TAG)` rather than a function?

Also wondering about the use of `UNIVERSAL_TAG`, it this link actually name spaced in the XML.

Reply:
I think this was just me not knowing the library as well a few months ago, not that I'm apprenticed by any means. 

I started off with a `Project` object in a REPL, and asked ... 
+ how do I get the file nodes from this `ClarityElement` object (inherited `WrappedXml`)? 
resolved by `xml_findall`
+ now, how do I use these file nodes to instantiate some s4-clarity `File` objects?
resolved by using our `LIMS` factory

Is the link actually name spaced in the XML?
Again, I was just iteratively processing all of this so it wouldn't have been strange for me to see it name spaced. On second inspection -- `file` is name spaced, where as `researcher` is not. From your questions, I'm guessing that this behaviour is atypical? Example:
```
<prj:project xmlns:udf="http://genologics.com/ri/userdefined" xmlns:ri="http://genologics.com/ri" xmlns:file="http://genologics.com/ri/file" xmlns:prj="http://genologics.com/ri/project" uri="https://qalocal/api/v2/projects/ADM36" limsid="ADM36">
<name>SampleTug-1857BfDl6Dv2aTLs</name>
<open-date>2021-11-10</open-date>
<researcher uri="https://qalocal/api/v2/researchers/1"/>
<file:file limsid="40-4102" uri="https://qalocal/api/v2/files/40-4102"/>
<file:file limsid="40-4101" uri="https://qalocal/api/v2/files/40-4101"/>
</prj:project>
```